src/main/scala/ru/gaptrakhmanov/telegram/helper/commands/wordgame/dto/UsersAnswers.scala

```diff
@@ -0,0 +1,7 @@
+package ru.gaptrakhmanov.telegram.helper.commands.wordgame.dto
+
+case class UsersAnswers(set: Set[String])
```

Comment:
My only suggestion here is to give a better name to the attribute `set`, maybe something like answers.